### ISSUE TITLE
Full-time (all-year-round) nodeschool

### ISSUE THREAD

**Issue by balupton** [nodeschool/discussions#402]
It's possible that I may be able to put together a physical school for teaching JavaScript and Node.js that will operate for extended periods of time.

I'm open to the location and the format, but as an initial brainstorm to get things rolling here's an alpha quality idea of what it could be:

- An office space able to house 10 people
- An event room able to house 30+ people
- 10 week paid training courses every 3 months
- Fortnightly trainings on Saturday; odd weeks will be node school; off weeks will be electives
- Open location for mentors to prepare their own workshops, practice their presentation skills
- Day passes for students for co-working
- Able to take and refer companies to relevant consultants

In other words, a physical hub for all things node.js. It could probably be accomplished in Sydney with @Rob-Morris's help, otherwise I'm open to suggestions. @elf-pavlik and co actually setup something like this (for general startup folk) in spain for free (house was donated, food was donated).

I'm really interested in setting this up, and if anyone has ideas for this, and if anyone is interested in this, and if anyone can help with this, post a comment.

/cc @damonoehlman @rvagg

**Comment by martinheidegger** [comment]
I take it the idea was nice but hasn't gone anywhere so I close this issue. But I will open an issue regarding organising a NodeSchool foundation (which share's similarities between Japan and UK btw.).
